lib/collections/accounts.js

```diff
@@ -0,0 +1,113 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+import { shopIdAutoValue } from "./helpers";
+import { Address } from "./address";
+import { Metafield } from "./metafield";
+
+/**
+ * Accounts Schemas
+ */
+
+export const Profile = new SimpleSchema({
```

Comment:
Sorry, I think I may have misread this because the comment says `Accounts Schema`. You can disregard that comment.